package.json

```diff
@@ -48,9 +49,11 @@
     "typescript": "^2.1.5"
   },
   "dependencies": {
+    "@types/react-router": "^3.0.6",
     "asap": "https://github.com/alloy/asap.git",
     "react": "^15.4.2",
     "react-relay": "https://github.com/alloy/relay/releases/download/v0.9.3/react-relay-0.9.3.tgz",
+    "react-router": "^3.0.2",
```

Comment:
Yeah basically `react-router` is only being used for the loyalty app. I agree that it'd be easier to have the app directly in force and keep reaction-force components separate. we may just end up doing just that.